### PR TITLE
MNTOR-2789 - Add exposure details aria-label to exposure card chevron

### PR DESCRIPTION
**Pull request by codemist**
# References: 
Jira: MNTOR-2789

# Checklist (Definition of Done)
- [ ] Localization strings (if needed) have been added.
- [ ] Commits in this PR are minimal and [have descriptive commit messages](https://chris.beams.io/posts/git-commit/).
- [ ] I've added or updated the relevant sections in readme and/or code comments
- [ ] I've added a unit test to test for potential regressions of this bug.
- [ ] Product Owner accepted the User Story (demo of functionality completed) or waived the privilege.
- [ ] All acceptance criteria are met.
- [ ] Jira ticket has been updated (if needed) to match changes made during the development process.
- [ ] Jira ticket has been updated (if needed) with suggestions for QA when this PR is deployed to stage.

### CODE REVIEW

File: src/app/components/client/exposure_card/ScanResultCard.tsx [hunk 1]
```diff
@@ -246,15 +246,12 @@ export const ScanResultCard = (props: ScanResultCardProps) => {
             className={styles.chevron}
             onClick={() => props.onToggleExpanded()}
             aria-expanded={props.isExpanded}
+            aria-label={`${l10n.getString("chevron-alt")} ${props.isExpanded ? l10n.getString("chevron-expanded") : l10n.getString("chevron-collapsed")}`}
```
Comment (Vinnl) [inline src/app/components/client/exposure_card/ScanResultCard.tsx:249]: As I understood the ticket, we don't need to manually add "Expanded" or "Collapsed" to this string - the screen reader will announce that by itself, thanks to the `aria-expanded` attribute. So currently when I use my screen reader, this announces "Exposure details collapsed collapsed".

(I'd also expect this label to be used as the `alt` for the image as we did before, as `aria-label` is a last resort for when there is no content, but I think in practice the effect will be the same.)

```suggestion
            aria-label={l10n.getString("chevron-alt")}
```

File: src/app/components/client/exposure_card/SubscriberBreachCard.tsx [hunk 2]
```diff
@@ -209,18 +209,14 @@ export const SubscriberBreachCard = (props: SubscriberBreachCardProps) => {
             // TODO: Add unit test when changing this code:
             /* c8 ignore next */
             onClick={props.onToggleExpanded}
+            aria-expanded={props.isExpanded}
+            aria-label={`${l10n.getString("chevron-alt")} ${props.isExpanded ? l10n.getString("chevron-expanded") : l10n.getString("chevron-collapsed")}`}
```
Comment (Vinnl) [inline src/app/components/client/exposure_card/SubscriberBreachCard.tsx:213]: (Same as above)

```suggestion
            aria-label={l10n.getString("chevron-alt")}
```

**Review by Vinnl — COMMENTED** [review]
I get the "Expanded" or "Collapsed" announcement twice now in my screen reader, unfortunately. Though probably a simple fix? (And would allow the removal of two strings?)

File: src/app/components/client/exposure_card/ScanResultCard.tsx (hunk 1 above)
Comment (Vinnl) [reply src/app/components/client/exposure_card/ScanResultCard.tsx:249]: You can just select the button with the same string. If you want to verify whether the button indicates expanded/collapsed state, you could e.g. do

```
expect(expandButton).toHaveAttribute("aria-expanded", "true")
```

**Review by flodolo — APPROVED** [review]

**Review by Vinnl — APPROVED** [review]
Thank you!

**Pull request merged**